config/defaults.conf

```diff
@@ -54,191 +60,225 @@ local_log_path = ${run_dir}"/log.log"
+do_pretrain = 1  // Train the shared sentence encoder model (and task-specific model parameters)
+                 // on the pretraining tasks in pretrain_tasks.
+do_target_task_training = 1  // After do_pretrain, train on the target tasks in target_tasks.
+do_full_eval = 1  // Evaluate on the tasks in target_tasks.
 
 // Related configuration
-load_model = 1  // If true, restore from checkpoint when starting do_pretrain. No impact on do_target_task_training.
-transfer_paradigm = "frozen" // How to do learning on top of the pretrained model
-                             // this option also affect the updating of downloaded parameters for models like ELMO/GPT/BERT
+load_model = 1  // If true, restore from checkpoint when starting do_pretrain. No impact on
+                // do_target_task_training.
+transfer_paradigm = "frozen" // How to use pretrained model parameters during target task training.
+                             // Applies to GPT, BERT, and pretrained sentence encoders.
                              // Options: "frozen", "finetune"
-                             // "frozen" will train the downstream models on fixed representations from the encoder model
-                             // "finetune" will update the parameters of the encoders models as well as the downstream models
-load_target_train_checkpoint = none  // If not "none", load the specified model_state checkpoint file when starting do_target_task_training.
-allow_untrained_encoder_parameters = 0  // Set for experiments involving random untrained encoders only. Allows do_target_task_training
-                                        // and do_full_eval to proceed without pretraining.
-allow_reuse_of_pretraining_parameters = 0  // Set to 1 to allow task models that were trained during pretraining from being
-                                           // reused in do_target_task_training. This may behave incorrectly if a run is stopped and
-                                           // restarted in do_target_task_training (issues #285, #290).
+                             //   "frozen" will train the downstream models on fixed 
+                             //   representations from the encoder model.
+                             //   "finetune" will update the parameters of the encoders models as 
+                             //    well as the downstream models.
+load_target_train_checkpoint = none  // If not "none", load the specified model_state checkpoint 
+                                     // file when starting do_target_task_training.
+allow_untrained_encoder_parameters = 0  // Set for experiments involving random untrained encoders 
+                                        // only. Allows do_target_task_training and do_full_eval 
+                                        // to proceed without pretraining.
+allow_reuse_of_pretraining_parameters = 0  // Set to 1 to allow task models that were trained 
```

Comment:
Slightly weird grammar here "allow task models...from being reused", probably want to change to "to be reused"